library/src/main/java/com/datatorrent/lib/io/fs/AbstractFileSplitter.java

```diff
@@ -53,6 +53,7 @@
    * This is a threshold on the no. of blocks emitted per window. A lot of blocks emitted
    * per window can overwhelm the downstream operators. This setting helps to control that.
    */
+  @NotNull
```

Comment:
It will because min constraint is 1. By default the field will be assigned 0. So the user will have to change the value.  Please don't change types